vms/secp256k1fx/camino_credential.go

```diff
@@ -0,0 +1,69 @@
+// Copyright (C) 2023, Chain4Travel AG. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package secp256k1fx
+
+import (
+	"bytes"
+	"encoding/json"
+	"errors"
+
+	"github.com/ava-labs/avalanchego/ids"
+	"github.com/ava-labs/avalanchego/utils"
+	"github.com/ava-labs/avalanchego/utils/crypto"
+)
+
+var errMSigNotUniqueOrSorted = errors.New("multisig aliases not sorted or unique")
+
+type CredentialIntf interface {
+	Signatures() [][crypto.SECP256K1RSigLen]byte
+	Verify() error
```

Comment:
maybe include Verifiable interface instead of Verify func?

Reply:
ok